src/sage/groups/cubic_braid.py

```diff
@@ -93,11 +93,7 @@
 from sage.rings.integer import Integer
 from sage.structure.unique_representation import UniqueRepresentation
 
-
-try:
-    from sage.libs.gap.element import GapElement
-except ImportError:
-    GapElement = ()
+from sage.libs.gap.element import GapElement
```

Comment:
are we not doing modularization anymore? (I don't think there's any discussion of this whatsoever? Usually decisions are made by voting on `sage-devel`, right. Except of course `mkoeppe` is no longer around to push this)

if we don't do modularization, it might be a good idea to delete all the `# needs sage.abc.def` comments as they cause some clutter. First by changing `sage-fixdoctests` to no longer add these.